src/main/java/core/basesyntax/ReportBuilder.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+public class ReportBuilder {
```

Comment:
I don't like how this class has 2 responsibilities - it parses CSV string and generates another CSV string. Do you think it would be better to split it into two classes - first would just parse CSV string and return parsed data in form of `String[][]`, the second class will generate CSV from this `String[][]` data.